db/migrate/20200814142059_create_enrollments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateEnrollments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :enrollments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :course, null: false, foreign_key: true
```

Comment:
I think you should add an index for these two fields for search.
And remove index for `user_id`:
```ruby
t.references :user, null: false, foreign_key: true, index: false
```
If you have an index on two columns (`user_id` and `course_id`), you shouldn't create an index for the first field (`user_id` in our case), because index on two columns will be suitable for users too, so it and will be used for them too.